powerauth-credential-server-sample/src/main/java/io/getlime/security/powerauth/lib/credentials/controller/SMSAuthorizationController.java

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright 2017 Lime - HighTech Solutions s.r.o.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.getlime.security.powerauth.lib.credentials.controller;
+
+import io.getlime.core.rest.model.base.request.ObjectRequest;
+import io.getlime.core.rest.model.base.response.ObjectResponse;
+import io.getlime.security.powerauth.lib.credentials.configuration.CredentialStoreConfiguration;
+import io.getlime.security.powerauth.lib.credentials.exception.SMSAuthorizationFailedException;
+import io.getlime.security.powerauth.lib.credentials.exception.SMSAuthorizationMessageInvalidException;
+import io.getlime.security.powerauth.lib.credentials.model.request.CreateSMSAuthorizationRequest;
+import io.getlime.security.powerauth.lib.credentials.model.request.VerifySMSAuthorizationRequest;
+import io.getlime.security.powerauth.lib.credentials.model.response.CreateSMSAuthorizationResponse;
+import io.getlime.security.powerauth.lib.credentials.repository.SMSAuthorizationRepository;
+import io.getlime.security.powerauth.lib.credentials.repository.model.entity.SMSAuthorizationEntity;
+import io.getlime.security.powerauth.lib.credentials.validation.CreateSMSAuthorizationRequestValidator;
+import org.joda.time.DateTime;
+import org.springframework.core.MethodParameter;
+import org.springframework.stereotype.Controller;
+import org.springframework.validation.BeanPropertyBindingResult;
+import org.springframework.validation.ValidationUtils;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+import java.math.BigDecimal;
+import java.security.SecureRandom;
+import java.util.Date;
+import java.util.UUID;
+
+/**
+ * Controller class which handles SMS OTP authorization.
+ *
+ * @author Roman Strobl, roman.strobl@lime-company.eu
+ */
+@Controller
```

Comment:
We are missing base RequestMapping annotation here, and we are not consistent with other controllers:

```java
@Controller
@RequestMapping("/api/auth/sms")
public class ...
```

Next calls then can be called just `/create`, etc.